uav_cloud: /tmp/uav_normals_for_exp01.ply
frontier_cloud_folder: 
#frontier_cloud:  
offset: [0.0,0.0,0.0]
output_folder: 
debug: False
save_pose_graph: True
correspondence_matching_method: graph
bls_feature_extraction_method: "canopy_map"
crop_frontier_cloud: False
icp_fitness_score_threshold: 0.85
